samples/wifi/radio_test/README.rst

```diff
@@ -0,0 +1,363 @@
+.. _wifi_radio_test:
+
+Wi-Fi: Radio test
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Wi-Fi Radio test sample demonstrates how to configure the Wi-Fi radio in a specific mode and then test its performance.
+The sample provides a set of predefined commands that allow you to configure the radio in the following modes:
+
+* Modulated carrier TX
+* Modulated carrier RX
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+To run the tests, connect to the development kit through the serial port and send shell commands.
+Zephyr's :ref:`zephyr:shell_api` module is used to handle the commands.
+
+You can then start running ``wifi_radio_test`` subcommands to set up and control the radio.
+See :ref:`wifi_radio_test_subcmds` for a list of available subcommands.
+
+In the `Modulated carrier RX` mode the `get_stats` subcommand can be executed to display the statistics.
+See :ref:`wifi_radio_test_stats` for a list of available statistics.
+
+User Interface
+**************
+
+``wifi_radio_test`` is the Wi-Fi radio test command and supports the following subcommands:
+
+.. _wifi_radio_test_subcmds:
+
+Wi-Fi radio test subcommands
+============================
+
+.. list-table:: Wi-Fi radio test subcommands
+   :header-rows: 1
+
+   * - Subcommand
+     - Argument
+     - Description
+   * - phy_calib_rxdc
+     - | 0 – Disable
+       | 1 – Enable
+     - Enable/Disable RX DC calibration.
```

Comment:
Should we spell out the abbreviations like DC, IQ, DPD..?

Reply:
I don't think it is necessary since the user is expected to be aware of this terminology to use this sample.